local-volume/provisioner/pkg/common/common.go

```diff
@@ -130,20 +145,12 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 // GetVolumeConfigFromMountedConfigMap gets volume configuration from the mounted configmap volume,
 func GetVolumeConfigFromMountedConfigMap() (map[string]MountConfig, error) {
```

Comment:
Is this still needed? Does `LoadProvisionerConfigs` cover this?

Reply:
no need, removing